critiquebrainz/frontend/views/moderators.py

```diff
@@ -0,0 +1,25 @@
+from hashlib import md5
+from flask import Blueprint, render_template
+from flask import current_app
+from critiquebrainz.db import users as db_users
+
+moderators_bp = Blueprint('moderators', __name__)
+
+
+@moderators_bp.route('/')
+def moderators():
+    admins = current_app.config['ADMINS']
+    moderators = db_users.get_many_by_mb_username(admins)
+    config_list = [moderator['display_name'] for moderator in moderators]
+    # list of MB users without CritiqueBrainz account
+    missing_users = list(set(admins) - set(config_list))
```

Comment:
No, I can't change change it that way, because I need to use `admins` list to compare it with information on users from db, and find out which users don't have CritiqueBrainz account.